src/lib/server/api/api-getters.ts

```diff
@@ -4,7 +4,7 @@ import { handleError } from '$lib/scripts/helper/errorHandler';
 
 async function getImages(
 	many: boolean = true,
-	height: number = 2000,
+	height: number = 1200,
```

Comment:
future suggesstion:

Height should be a value that would look good visually on all screens
```suggestion
	height: number = 2000,
```